### ISSUE TITLE
Cannot invoke local for container-image-based lambda

### ISSUE THREAD

**Issue by Holi0317** [serverless/serverless#12092]
### Are you certain it's a bug?

- [X] Yes, it looks like a bug

### Is the issue caused by a plugin?

- [X] It is not a plugin issue

### Are you using the latest v3 release?

- [X] Yes, I'm using the latest v3 release

### Is there an existing issue for this?

- [X] I have searched existing issues, it hasn't been reported yet

### Issue description

I'm trying to convert one of my lambda from zip deployment to docker-based deployment. Following the guide in https://www.serverless.com/blog/container-support-for-lambda, I've come up with a `Dockerfile` and `serverless.yml` for it.

When testing the lambda locally with `sls invoke local`, it failed with "Local invocation of lambdas pointing AWS ECR images is not supported" error.

There are existing code in https://github.com/serverless/serverless/blob/47752842e00b6966159deadad67b9d15d305d91b/lib/plugins/aws/invoke-local/index.js for interacting with docker. Is it possible to add docker invoke support in it?

### Service configuration (serverless.yml) content

```yaml
service: testsrv

provider:
  name: aws
  runtime: nodejs14.x
  stage: dev
  region: eu-west-1
  memorySize: 3008

  versionFunctions: false

  ecr:
    scanOnPush: true
    images:
      baseimage:
        platform: "linux/amd64"
        path: "."
        file: "Dockerfile"
        buildArgs:
          DEV: "${param:build_arg_dev, '1'}" # CircleCI will set this to 0

functions:
  allocation:
    image:
      name: "baseimage"
      command: ["functions/allocation.handler"]
    timeout: 300
```

### Command name and used flags

sls invoke local --docker --verbose -f allocation -s uat

### Command output

```shell
Environment: darwin, node 18.16.1, framework 3.33.0 (local) 3.33.0v (global), plugin 6.2.3, SDK 4.3.2
Credentials: Local, "default" profile
Docs:        docs.serverless.com
Support:     forum.serverless.com
Bugs:        github.com/serverless/serverless/issues

Error:
Local invocation of lambdas pointing AWS ECR images is not supported
```
```


### Environment information

```shell
Running "serverless" from node_modules
Framework Core: 3.33.0 (local) 3.33.0 (global)
Plugin: 6.2.3
SDK: 4.3.2
```
```

**Comment by sc0ttdav3y** [comment]
As a workaround, those trying to run local docker lambdas can follow the official AWS instructions here:
https://docs.aws.amazon.com/lambda/latest/dg/images-test.html

TL;DR:

```
docker run --platform linux/amd64 -p 9000:8080 <your ECS image URI>
```

and then

```
curl "http://localhost:9000/2015-03-31/functions/function/invocations" -d '{"payload":"hello world!"}'
```

**Comment by rashedoumar** [comment]
what if i need to invoke it without HTTP request, like local invoke :
``serverless invoke local -f function``
